src/entities/session/model/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './api';
```

Comment:
why do you export api in sessionModel? from session model you have to export existing effects, we are not working with api methods directly